mautic/mautic
# Fix api limit issue

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | Yes
| New feature/enhancement? (use the a.x branch)      | No
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | No
| Automated tests included?              | Yes 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Limit parameter was not working as expected, Instance have 150 emails, with the limit set to 200, api return 105 records only.


#### Steps to test this PR:



1. Create 3-4 segment
2. Create 3-4 segment email and attach with all 3-4 segments in each email or some email
3. Now fetch email list using limit via api (/api/emails?limit={limit})
4. Api should return expected list of emails.
